css/index.css

```diff
@@ -112,6 +148,117 @@ p {
     justify-content: center;
 }
 
```

Comment:
The .box class can take care of all of the styling of the child boxes and then you can just change the color in each box.